src/redux/categories/categorySlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: '',
+};
+
+const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    checkStatus: (state) => ({
+      ...state,
+      status: 'Under construction',
+    }),
+  },
+});
+
+// console.log(categorySlice);
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉